src/cli.ts

```diff
@@ -21,6 +21,7 @@ async function main() {
 		output,
 		tableProps,
 		tmp_jyc,
+		custom_decorators,
```

Comment:
```suggestion
		tmp-jyc,
		custom-decorators,
```

i think we should rethink multiword arg case
![image](https://github.com/Yedidyar/remult-cli/assets/42849252/16d10300-3111-490e-98a1-0352dea2fc94)
![image](https://github.com/Yedidyar/remult-cli/assets/42849252/6134e945-99b4-4613-b50c-a9e005d7e064)